src/Fields/Field.php

```diff
@@ -172,7 +185,7 @@ public function isListable()
         return (bool) $this->get('listable');
     }
 
-    public function isVisible()
+    public function isVisibleOnListing()
```

Comment:
The reason is because it is confusing to have both `visibility()` and `isVisible()`, when the latter was only ever used for listings (never publish form state).

- `visibility()` gets the visibility status for publish forms (ie. `visible`, `read_only`, `hidden`, and soon `computed`)
- `isVisibleOnListing()` is only ever used for listings, and checks the field's `listable` config determine whether the field should be shown on listings.

Re: Breaking change, that's fair, though we never documented it anywhere. I suppose there may be an addon that could reference it. If that's the case, maybe I add `isVisible()` as an alias with `@deprecated` docblock for 3.4/4.0?

Reply:
Yeah, the deprecated alias will work.